app/main/posts/views/mode-context-form-filter.directive.js

```diff
@@ -27,9 +27,38 @@ function ModeContextFormFilter($scope, FormEndpoint, PostEndpoint, TagEndpoint,
     $scope.unknown = [];
 
     activate();
-
-    $scope.$watch('filters', function () {
-        getPostStats(PostFilters.getFilters()).$promise.then(updateCounts);
+    /**
+     * TODO this needs refactoring about refactoring the  qenabled/reactivefilters situation
```

Comment:
I think this is ok for now, but its definitely fairly fragile.

Reply:
Agree.